src/app/content/components/Toolbar/PracticeQuestionsButton.tsx

```diff
@@ -25,24 +45,29 @@ const PracticeQuestionsText = styled.span`
 
 // tslint:disable-next-line:variable-name
 const PracticeQuestionsButton = () => {
-  const dispatch = useDispatch();
   const isEnabled = useSelector(practiceQuestionsEnabled);
   const trackOpenClose = useAnalyticsEvent('openClosePracticeQuestions');
   const hasPracticeQs = useSelector(hasPracticeQuestions);
+  const { book, page }: {book: any; page: any} = useSelector(bookAndPage);
 
-  if (!isEnabled || !hasPracticeQs) { return null; }
+  if (!isEnabled || !hasPracticeQs || !book || !page) { return null; }
 
-  const openPracticeQuestionsSummary = () => {
-    dispatch(openPracticeQuestions());
-    trackOpenClose();
+  const search = {
+    modal: 'PQ',
+    query: undefined,
```

Comment:
I think you shouldn't have to specify the `query` and there should be a `constant` with a `PQ` value